cv-data-service-library/src/main/java/com/trihydro/library/helpers/TimGenerationHelper.java

```diff
@@ -817,13 +817,14 @@ private List<ResubmitTimException> sendTim(WydotTravelerInputData timToSend, Tim
         return exceptions;
     }
 
-    private int getMsgCnt(int currentCnt) {
-        if (currentCnt == 127) {
+    protected int getNextMessageCount(int currentCount) {
+        if (currentCount == 127) {
+            // reset to 1
             return 1;
         }
```

Comment:
chore(blocking): Ensure that we reset the count back to 1 permanently. Without the suggested change, we will return `1` each time we call `getNextMessageCount` after we hit the `127` cap. I don't think we want that. The suggested change will reset the counter to 1 and the next time this method is called we'll continue to increment the count
```suggestion
    protected int getNextMessageCount(int currentCount) {
        if (currentCount == 127) {
            // reset to 1
            currentCount = 1;
            return currentCount;
        }
```

Reply:
Updated this to modify the passed value and also simplified the method a bit!